src/ui/events.js

```diff
@@ -12,10 +12,27 @@ import type LngLat from '../geo/lng_lat';
 /**
  * `MapMouseEvent` is the event type for mouse-related map events.
  * @extends {Object}
+ * @example
+ * // The `click` event is an example of a `MapMouseEvent`.
+ * // Set up an event listener on the map.
+ * map.on('click', function(e) {
+ *   // The event object (`e`) contains information like the
```

Comment:
```suggestion
 *   // The event object (e) contains information like the
```

i would maybe remove the backticks here since they wont render as a code block.